pkg/load/agents/configAgent.go

```diff
@@ -226,7 +244,7 @@ func (a *configAgent) loadFilenameToConfig() error {
 		a.lock.Lock()
 		defer a.lock.Unlock()
 		startTime := time.Now()
-		configs, err := config.LoadByOrgRepo(a.configPath)
+		configs, err := config.LoadByOrgRepo(filepath.Join(a.configPath, a.org, a.repo))
```

Comment:
Didn't know `Join` ignores empty strings.  Nice.

Reply:
I checked also that! :)